src/app/components/form/form.component.ts

```diff
@@ -121,6 +117,16 @@ export class FormComponent implements OnInit {
     return selectedItems.length ? selectedItems : null;
   }
 
+  private displayDataFromParent() {
+
+    if (this.form['domain'] === '') {
+      this.alertService.error('Domain name required');
```

Comment:
A problem with this solution is that we always get the message in English, don't we? How can we have it translatable to the supported languages?

Reply:
The system is already in place. We need to put the translation in the lang files (src/assets/i18n/[fr/en/sv]) with the english text as the json key. I started this work for french but i'm not very confident with my Swedish :).
I create an issue for that #44

Reply:
What do you mean by "the system is already in place"? This is new code that has not yet been merged to develop branch. We are still discussing it.

Reply:
The code bellow send a string (here 'Domain name required') to the error method of the alert service. The alert service display the text with the proper settings (color red for error, orange for warning etc.) and also try to translate the text. If the translation is not define, it keep the string (here 'Domain name required') and display it. To see this system in action, switch to french and try to run a test without domain name. 

Yes, i will add the English translation in a separate PR.